src/eva/utilities/config.py

```diff
@@ -56,15 +56,15 @@ def get(self, key, default=None):
 # --------------------------------------------------------------------------------------------------
 
 
-def get(dict, logger, key, default=None):
+def get(dict, logger, key, default=None, abort_on_failure=True):
 
     if default is None:
 
         if key in dict:
 
             return dict.get(key)
 
-        else:
+        elif abort_on_failure:
```

Comment:
it may be good to have a warning print that the key is missing and the default is used, but I won't ask that be done here unless others want that.

Reply:
I'll go ahead and add a warning then.